### DISCUSSION TITLE
Space separated props value in props method

### DISCUSSION THREAD

**Discussion by michelemoretti** [zauberzeug/nicegui#589] [Q&A] [answered]
In case the quasar component has a prop value that is composed of two words that are separated by a space like [the anchor props in this component](https://quasar.dev/vue-components/tooltip#qtooltip-api) is there a way to pass it to the Element props method since the props need to be space separated themselves?

**Comment by falkoschindler** [comment, marked as answer]
Sure, you can wrap multiple words in double quotes:

```py
with ui.label('Note how the tooltip...').classes('w-64 h-48 bg-gray-200'):
    ui.tooltip('is anchored at the bottom right corner.').props('anchor="bottom right"')
```
